Gradle.md

```diff
@@ -19,29 +19,29 @@ Note how to deal with the eventuality of facing filename too long errors [here](
 
 ### User permissions
 
-Experiment builds running locally require different permissions than those that run in CI.
-Note that while not all permissions are required, they're still strongly recommended for the best possible experience.
+The Develocity Build Validation Scripts involve running builds on your local machine and CI environment.
+The builds run on your local machine require a different set of Develocity user permissions than those run in your CI environment.
 
-Your Develocity user should be granted the following permissions:
+#### Permissions required for your Develocity user
 
-| User permission                 | Reason                                                                                                                                      | Required              |
-|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|-----------------------|
-| Publish build scans             | Analyzing the experiment results relies on build scans published to Develocity.                                                             | Yes                   |
-| Read build cache data           | Reading from the remote build cache.                                                                                                        | Only for experiment 5 |
-| View build scans and build data | Build scans can be used to perform a deeper analysis of the experiment results.                                                             | No                    |
-| Access build data via the API   | Used to retrieve the summarized experiment results. See [Authenticating with Develocity](#authenticating-with-develocity) for more details. | No                    |
+| User permission                 | Reason                                                                                                                                      |
+|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| Publish build scans             | Analyzing the experiment results relies on build scans published to Develocity.                                                             |
+| Read build cache data           | Reading from the remote build cache.                                                                                                        |
+| View build scans and build data | Build scans can be used to perform a deeper analysis of the experiment results.                                                             |
+| Access build data via the API   | Used to retrieve the summarized experiment results. See [Authenticating with Develocity](#authenticating-with-develocity) for more details. |
```

Comment:
`Retrieving the published build data from Develocity to summarize the experiment results.`